File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Java/src/test/java/simulacionParking/models/ParkingTest.java
package simulacionParking.models;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class ParkingTest {

    @Test
    void getSizeRowTest() {
        Vehiculo[][] matriz = new Vehiculo[3][4];
        Parking parking = new Parking(3, 4, matriz);

        int sizeRow = parking.getSizeRow();
        assertEquals(3, sizeRow);
    }

    @Test
    void getSizeColumnTest() {
        Vehiculo[][] matriz = new Vehiculo[3][4];
        Parking parking = new Parking(3, 4, matriz);

        int sizeColumn = parking.getSizeColumn();
        assertEquals(4, sizeColumn);
    }

    @Test
    void getParkingMatrixTest() {
        Vehiculo[][] matriz = new Vehiculo[3][4];
        Parking parking = new Parking(3, 4, matriz);

        assertEquals(matriz, parking.getParkingMatrix());
    }

}

File: aparcamiento/AMarotoChivite-aparcamiento/AMarotoChivite-aparcamiento-Kotlin/src/test/kotlin/simulacionParking/models/ParkingTest.kt
package simulacionParking.models

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

internal class ParkingTest {
    @get:Test
    val sizeRowTest: Unit
        get() {
            val matriz = Array(3) { arrayOfNulls<Vehiculo>(4) }
            val (sizeRow) = Parking(3, 4, matriz)
            Assertions.assertEquals(3, sizeRow)
        }

    @get:Test
    val sizeColumnTest: Unit
        get() {
            val matriz = Array(3) { arrayOfNulls<Vehiculo>(4) }
            val (_, sizeColumn) = Parking(3, 4, matriz)
            Assertions.assertEquals(4, sizeColumn)
        }

    @get:Test
    val parkingMatrixTest: Unit
        get() {
            val matriz = Array(3) { arrayOfNulls<Vehiculo>(4) }
            val parking = Parking(3, 4, matriz)
            assertEquals(matriz, parking.getParkingMatrix())
        }
}